### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jfairchild**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/CrisisTextLine/CrisisTrends.svg?branch=v2)](https://travis-ci.org/CrisisTextLine/CrisisTrends)
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FCrisisTextLine%2FCrisisTrends%2Fbadge%3Fref%3Dv2&style=for-the-badge)](https://actions-badge.atrox.dev/CrisisTextLine/CrisisTrends/goto?ref=v2)
```
Comment (jfairchild) [inline README.md:1]: adding a cool badge for Github Actions

File: app/scripts/config.js [hunk 2]
```diff
@@ -18,7 +18,7 @@
     },
     messages: {
       // count must be an integer
-      fallbackMessageCount: 160174788,
+      fallbackMessageCount: 160475788,
```
Comment (kuikkick) [inline app/scripts/config.js:21]: What determines this number?

**Review by kuikkick — APPROVED** [review]
LGTM!

File: app/scripts/config.js (hunk 2 above)
Comment (jfairchild) [reply app/scripts/config.js:21]: The API Gateway endpoint. https://github.com/CrisisTextLine/CrisisTrends#message-count

Comment (jfairchild) [reply app/scripts/config.js:21]: https://github.com/CrisisTextLine/CrisisTrends/blob/v2/app/scripts/sections/message-count.js#L5

here's the code to dynamically update the count.

Comment (kuikkick) [reply app/scripts/config.js:21]: Thank you!! This question and answer should go in Stack Overflow XD

**Pull request merged**